spec/rollbar_spec.rb

```diff
@@ -1863,6 +1893,16 @@ def backtrace
         expect(thread.priority).to eq(custom_priority)
         thread.join
       end
+
+      it "shows 'Rollbar::Delay::Thread' on the log" do
```

Comment:
I found that ActiveJob test does not exist on rollbar_spec
but it shouldn't be a problem I think